src/core/namespace.c

```diff
@@ -3855,6 +3856,27 @@ int refresh_extensions_in_namespace(
         return 0;
 }
 
+void bpf_delegate_to_string(uint64_t u, char *s) {
+        if (u == ~0ULL)
+                strcpy(s, "any");
+        else
+                sprintf(s, "0x%lx", u);
+}
+
+int bpf_delegate_from_string(const char *s, uint64_t *u) {
+        int r;
+
+        r = safe_atou64(s, u);
+        if (r < 0) {
+                if (streq(s, "any"))
+                        *u = ~0ULL;
```

Comment:
Not a matter, but I'd like to check with `"any"` first, then call `safe_atoux64()` (or `safe_atou64`).